release/release_tests.yaml

```diff
@@ -4832,6 +4619,8 @@
     script: RAY_DATASET_PUSH_BASED_SHUFFLE=1 python dataset/sort.py --num-partitions=1000 --partition-size=1e9 --shuffle
     wait_for_nodes:
       num_nodes: 20
+    
+    type: sdk_command
```

Comment:
why sdk?

Reply:
update: saw the comment above.

Reply:
this was the old default, so this test has not been migrated